doc/start/hardware-recommendations.rst

```diff
@@ -127,23 +138,25 @@ at $150.00 has a cost of $0.05 per gigabyte (i.e., $150 / 3072 = 0.0488). In the
 foregoing example, using the 1 terabyte disks would generally increase the cost
 per gigabyte by 40%--rendering your cluster substantially less cost efficient.
 
-.. tip:: Running multiple OSDs on a single disk--irrespective of partitions--is 
-   **NOT** a good idea.
+.. tip:: Running multiple OSDs on a single SAS / SATA drive -- irrespective of
+   partitions -- is **NOT** a good idea.  NVMe drives, however, can realize
+   improved performance by being split into two more more OSDs.
 
 .. tip:: Running an OSD and a monitor or a metadata server on a single 
-   disk--irrespective of partitions--is **NOT** a good idea either.
+   disk -- irrespective of partitions --is **NOT** a good idea either.
```

Comment:
I think there should be a space between the hyphen and the "i" in "is".

Reply:
Actually I realize that the awkward "irrespective" doesn't add meaning so I elided.